src/apps/workspaces/crud/user_applet_access.py

```diff
@@ -498,6 +498,12 @@ async def get_workspace_respondents(
                     SubjectSchema.id,
                     text("'subject_tag'"),
                     SubjectSchema.tag,
+                    text("'subject_first_name'"),
+                    SubjectSchema.first_name,
```

Comment:
This was removed by SS few weeks ago, should check with them why

Reply:
Thanks I will check with them, but we need it for [M2-6459](https://mindlogger.atlassian.net/browse/M2-6459) anyway, so it will have to go back

Reply:
Ok, in that case I think we can resolve this